docker/4.4.0-alpine/Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM java:openjdk-8-jre-alpine
```

Comment:
is it possible to have a template and generate Docker files based on different JDK?